### PR TITLE
Add April community update

### PR DESCRIPTION
**Pull request by sims1253**

### CODE REVIEW

File: _posts/2017-04-18-community-team-update.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+title: "April Community Team Update"
+categories: coala
+tags: [community team]
```
Comment (meetmangukiya) [inline _posts/2017-04-18-community-team-update.md]: Don't you need to enclose it in quotes? Ignore if not ;)

File: _posts/2017-04-18-community-team-update.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+Long long time ago...
+
+So what have we been up to?
+And where were all those monthly updates?
+
+
+GSoC 2017
+---------
+
+So yea. coala is in. We are our own organization and might even serve as an
```
Comment (meetmangukiya) [inline _posts/2017-04-18-community-team-update.md]: Yes or do you mean Yeah?

File: _posts/2017-04-18-community-team-update.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+I won't mentor anyone as we feel like a designated admin with no student ties
+might be a good idea if any conflict come up.
+This also keeps me available as a backup mentor if everything else fails and
+generally let's me keep eyes on the hoard with less distraction.
+
+
+### Projects Page
+
+So I talked about the project page in the first post.
+Go visit <coala.io/projects> and see for yourself.
```
Comment (meetmangukiya) [inline _posts/2017-04-18-community-team-update.md]: Use projects.coala.io instead. Upto you though

File: _posts/2017-04-18-community-team-update.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+
+So what is coming up?
+
+- There will be a lot of GSoC work for sure.
+- I just wrote the intro for our
+  [Call for Chefs](http://blog.solyoution.io/coala/coala-call-for-chefs.html)
+  cooking competition,
+  that essentially was born over a ramen pep talk with ASNEL half a year ago.
+- I am updating our ideas page and will try to get more stuff into the works
+- We will hold our first maintainer meeting tomorrow. The idea is quite old but
```
Comment (meetmangukiya) [inline _posts/2017-04-18-community-team-update.md]: This is relative, be sure to change this if it is not merged today :P

File: _posts/2017-04-18-community-team-update.md (hunk 2 above)
Comment (hemangsk) [inline _posts/2017-04-18-community-team-update.md]: @meetmangukiya yea is also a proper word.

**Comment by meetmangukiya** [comment]
+1

File: _posts/2017-04-18-community-team-update.md (hunk 1 above)
Comment (sims1253) [inline _posts/2017-04-18-community-team-update.md]: I think without works as long as you only have one tag :D

**Comment by sims1253** [comment]
@rultor merge

**Pull request merged**